python_benchmarks/global_params.py

```diff
@@ -2,6 +2,37 @@
 from typing import Union, List, Tuple
 from nvfuser import DataType
 from .core import DEVICE_PROPERTIES
+import numpy as np
+import os
+
+# Model Parameters from LLMs (GPT2/3, PaLM, LLama)
+
+# Embedding size: d_model, d_ff = 4 * d_model
+D_MODEL_MIN = 768
+D_MODEL_MAX = 18432
+
+# Actual d_model sizes seen in models.
+D_MODEL_SIZES = [
+    768,
+    1024,
+    1280,
+    1536,
+    1600,
+    2048,
+    2560,
+    4096,
+    5120,
+    5140,
```

Comment:
Do you know where 5140 came from?  I think Liqiang discovered it was a typo.

Reply:
GPT-3 13B model.

Reply:
Since you raised this question, do you two think it's useful to annotate (in code comment) which model variation a size comes from?

Reply:
Done